CLAUDE.md

```
@@ -0,0 +1,118 @@
+| Install deps | `npm install` |
+| Build | `npm run build` (runs `tsgo --project tsconfig.json`) |
+| Lint | `npm run lint` (runs `oxlint --type-aware && oxfmt --check`) |
+| Auto-format | `npm run format` |
+| Test | `npm test` (builds, then `node --test '__tests__/*.ts'`) |
+| Coverage | `node --test --experimental-test-coverage --test-reporter=spec '__tests__/*.ts'` |
+
+All three "quality gates" (build, lint, test) must stay green.
+
+## Architecture at a glance
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)